detekt-rules-naming/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/naming/BooleanPropertyNaming.kt

```diff
@@ -79,7 +79,7 @@ class BooleanPropertyNaming(config: Config = Config.empty) : Rule(config) {
         val description = "Boolean property name should match a $allowedPattern pattern."
         return CodeSmell(
             issue,
-            Entity.from(declaration),
+            Entity.from(declaration.nameIdentifier ?: declaration),
```

Comment:
```suggestion
            Entity.atName(declaration),
```

It does exactly the same but it gives a bit more sematics.

Reply:
Whoops, I knew I was missing something but could figure out why it looked weird.

Reply:
Automation: https://github.com/detekt/detekt/pull/5444